### PR TITLE
Design/ Homepage, DetailsPage, EditPage and AddPage

### PR DESCRIPTION
**Pull request by AljoschaNy**

### CODE REVIEW

File: frontend/src/components/form/TextInput.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {TextInputType} from "../../types/types.ts";
+
+function TextInput(props:Readonly<TextInputType>) {
```
Comment (makitango) [inline frontend/src/components/form/TextInput.tsx:4]: 1/2
```suggestion
export default function TextInput(props:Readonly<TextInputType>) {
```

File: frontend/src/components/form/TextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                    value={props.value}
+                    onChange={(event:React.ChangeEvent<HTMLInputElement>) => props.onChange(event)}
+                />
+            </label>
+            <br/>
+        </div>
+    )
+}
+
+export default TextInput;
```
Comment (makitango) [inline frontend/src/components/form/TextInput.tsx:22]: 2/2
```suggestion
```
As above: I would consolidate the rows and get rid of the empty row, but it's optional

File: frontend/src/components/svg/CheckBold.tsx
Comment (makitango) [inline frontend/src/components/svg/CheckBold.tsx:1]: As above, I would consolidate the many lines into one export default function xyz.

File: frontend/src/components/svg/CrossBold.tsx
Comment (makitango) [inline frontend/src/components/svg/CrossBold.tsx:1]: As above, I would consolidate the many lines into one export default function xyz.

File: frontend/src/components/svg/EditIcon.tsx
Comment (makitango) [inline frontend/src/components/svg/EditIcon.tsx:1]: As above, I would consolidate the many lines into one export default function xyz.

File: frontend/src/components/svg/RightArrow.tsx
Comment (makitango) [inline frontend/src/components/svg/RightArrow.tsx:1]: As above, I would consolidate the many lines into one export default function xyz.

File: frontend/src/components/footer/FooterFormPages.tsx
Comment (makitango) [inline frontend/src/components/footer/FooterFormPages.tsx:1]: FooterFormPages needs to be exported, right? Also, using uppercase is meant to be used for default functions.

File: frontend/src/pages/DetailsPage.tsx
Comment (makitango) [inline frontend/src/pages/DetailsPage.tsx:1]: Does DetailsPage not need to be exported and imported into the root file for routing?

File: frontend/src/pages/EditPage.tsx
Comment (makitango) [inline frontend/src/pages/EditPage.tsx:1]: Same as with DetailsPage above

File: frontend/src/components/footer/FooterFormPages.tsx
Comment (AljoschaNy) [reply frontend/src/components/footer/FooterFormPages.tsx:1]: What do you mean exactly? The FooterFormPages component has been exported as the default

**Review by makitango — APPROVED** [review]

**Pull request merged**